src/osd/OSD.cc

```diff
@@ -1901,6 +1901,17 @@ bool OSDService::try_finish_pg_delete(PG *pg, unsigned old_pg_num)
   return osd->try_finish_pg_delete(pg, old_pg_num);
 }
 
+
+int64_t OSDService::get_pg_delete_cost(int64_t cost)
```

Comment:
Do you think there would be any other consumer of `OSDService::get_pg_delete_cost()`? If not, this can be moved into `OSDService::queue_for_pg_delete()` as a lambda function similar to what's done for recovery.

Reply:
Yeah it won't be used anywhere else, I'll make it a lambda function.